### PR TITLE
Add BRATS expert model to support MRI image

### PR DESCRIPTION
**Pull request by mingxin-zheng**
This PR
- adds BRATS for brain MRI segmentation.
- fix markdown link linting #53

### CODE REVIEW

File: m3/demo/experts/expert_monai_brats.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            output_dir,
+            modality="MRI",
+            slice_index=slice_index,
+            axis=2,
+            image_filename=get_slice_filenames(img_file[0], slice_index),
+            label_filename=seg_image,
+        )
+        transforms({"image": img_file[0], "label": seg_file})
+        # TODO: find the right label mapping
+        text_output = f"The results are <segmentation>. The colors in this image describe\nyellow and red: tumor core, only yellow: enhancing tumor, all colors: whole tumor\n"
```
Comment (mingxin-zheng) [inline m3/demo/experts/expert_monai_brats.py:119]: Hi @holgerroth , I copied these from 
https://github.com/Project-MONAI/VLM/blob/acdfcf8389205621472c72ebcf37fccd4367172b/m3/data_prepare/experts/expert_utils.py#L98

But I'm unclear what classes does the output label 1, 2, 3?, 4 correspond .

Comment (holgerroth) [reply m3/demo/experts/expert_monai_brats.py:119]: The model only cares about the color overlay. That's described in the prompt above.

**Review by holgerroth — APPROVED** [review]
Looks good to me.

File: m3/demo/experts/expert_monai_brats.py (hunk 1 above)
Comment (holgerroth) [reply m3/demo/experts/expert_monai_brats.py:119]: The output should match this description https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation#output

Comment (mingxin-zheng) [reply m3/demo/experts/expert_monai_brats.py:119]: Thanks @holgerroth .

Checking the [official data description](https://www.med.upenn.edu/sbia/brats2018/data.html) 

It seems that the following index should be used to produce:
0: background
1: necrotic and no-enhancing tumor core (NCR/NET), which should be counted TC and WT subregion
2: peritumoral edema (ED), which is counted only as WT subresion
4: GD-enhancing tumor (ET), which should be counted under ET, TC, WT subregion

ET: Enhancing Tumor, WT: Whole Tumor, TC: Tumor Core.

I will change this PR to incorporate the right mapping

**Pull request merged**